configure.ac

```diff
@@ -1954,6 +1967,11 @@ fi
 case $add_players in
   *omxplayer*)
       XB_ADD_PLAYER([OMXPLAYER], [omxplayer])
+      AC_DEFINE([TARGET_RASPBERRY_PI],[1],["Define to 1 for RASPBERRY_PI"])
```

Comment:
But you could build with this changes with the same configure flags on both platforms.

However that is the revert that I get on my changes:
On https://sources.gentoo.org/cgi-bin/viewvc.cgi/gentoo-x86/media-tv/kodi/ChangeLog?view=markup
29 Mar 2015; Mike Frysinger <vapier@gentoo.org> kodi-9999.ebuild,
Revert USE=raspberry-pi as this should not be a USE flag.

Use flags here are for selectable capabilities for a package

I'm not sure if with this changes on the configure I can get the changes on the main repository for kodi to be build on raspberry-pi but I can fight a little to get that in